ui/admin/app/components/form/user/index.hbs

```diff
@@ -52,7 +53,10 @@
 
   {{#if (can 'save model' @model)}}
     <form.actions
-      @enableEditText={{t 'actions.edit-form'}}
+      @ariaDescribedBy={{t
```

Comment:
I'm not sure I understand why we linked the ID of the form itself, what text are we trying to descriptively add to this by using `aria-describedby`? It also seems weird to use a translated text as the ID when the element itself should have the translated text

Reply:
I pinged you in the JIRA ticket comment but adding reference here for good measure.

(taking a screenshot so it has more context versus copy paste):

<img width="1650" height="1092" alt="CleanShot 2025-09-10 at 09 25 10@2x" src="https://github.com/user-attachments/assets/f3e319ed-6f11-4c8e-8dd4-4167d76d140e" />

https://hashicorp.atlassian.net/browse/FE-112?focusedCommentId=828913

Reply:
Going off @MelSumner thoughts in the JIRA ticket, I set the form ID value to be `user-details` versus `users` since to me it seems to make better sense that the form is specific to the `user-details`.  `Users` did not seem to make sense to me but maybe I am misunderstanding the purpose here.  

> It also seems weird to use a translated text as the ID when the element itself should have the translated text

@ZedLi from what I am getting here, it sounds like the form ID value should be semantic versus the ember generated ID.  Maybe Melanie could provide some clarity here for both of us.  I'm not all that familiar with `aria-describedby` prior to working on this.

Reply:
Right, I think all those choices make sense but it sounds like we just need to pick one choice? It seems we could leave the button as `Edit form` if we're adding one of the aria attributes. 

With regards to the current implementation with 3, I was specifically pointing out the fact that it's currently pointing to the form ID instead of something with descriptive text (which melanie suggests as the form title):

> where the value is the ID of the form title element (in this case, the element with the text “Users”)

Though that looks like it has some extra text which would pulled in as part of the helper doc link text.